### PR TITLE
Vendor Kubernetes v1.19.0-rc.4 and fix klog tmp dir issue

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind dependency-change

#### What this PR does / why we need it:
This also allows to use klog/v2 which is able to discard its log
outputs.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/crio/main.go [hunk 1]
```diff
@@ -97,18 +96,8 @@ scope of the CRI.
...
-
-	if err := klogFlagSet.Set("logtostderr", "false"); err != nil {
-		fmt.Fprintf(os.Stderr, "unable to set logtostderr for klog: %v\n", err)
-	}
-	if err := klogFlagSet.Set("alsologtostderr", "false"); err != nil {
-		fmt.Fprintf(os.Stderr, "unable to set alsologtostderr for klog: %v\n", err)
-	}
-
+	// Configure klog to not write any output
+	klog.LogToStderr(true)
```
Comment (saschagrunert) [inline cmd/crio/main.go:100]: @vrothberg I guess klog/v2 solves our issue here, right?

**Comment by TomSweeneyRedHat** [comment]
LGTM
and happy green test buttons

File: cmd/crio/main.go (hunk 1 above)
Comment (haircommander) [reply cmd/crio/main.go:100]: :sunglasses: FINALLY

**Pull request merged**